### PR TITLE
Milestone: Restaurant Page

### PR DESCRIPTION
**Pull request by newhorizon-tech**
Concepts used in the project: 

- DOM manipulation 
- Event Listeners
- ES6 Modules 
- ES6 Functions 
- Bundling Modules  
- Asset Management

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant-homepage",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (BrittanyBlake) [inline package.json]: - [x] You are missing the necessary run common commands like `build` and `development`. 
screenshot from pathwright 👇🏾 
Screen Shot 2021-03-02 at 5 56 43 PM
```suggestion
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @newhorizon-tech! 👋 
You have done an amazing job with this project!! 🤩  It looks good, works well and embedding the map was a great choice 🏆 There are just a few issues that need to be resolved before you can move on: ⬇️ 

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by newhorizon-tech** [comment]
Added common commands (dev, build, start) to package.json

**Review by arslanbisharat — APPROVED** [review]
# Approved 🥇 

Thank you for submitting this project. You have done a great job!

- [x] You have a descriptive Pull Request 👏 
- [x] You have a descriptive Readme. 👏 
- [x] Your commit messages are very descriptive too. I am very happy to see that.  👏 
- [x] No Linter error found. 👏 
- [x] Met Min. requirements. 

I am very happy to review your project. Looking forward to amazing new projects from your side. 

**Best,
Arslan Bisharat**
_You can always contact me on slack at @arslanbisharat 📞_

**Pull request merged**